backend/src/packages/orders/order.repository.ts

```diff
@@ -196,6 +267,30 @@ class OrderRepository implements Omit<IRepository, 'find'> {
 
     return Boolean(item);
   }
+
+  public async getTotal(
```

Comment:
```suggestion
  public async getUsersTotal(
```
we probably need such methods for driver and for business